services/beaconchain/src/main/java/tech/pegasys/teku/services/beaconchain/BeaconChainController.java

```diff
@@ -1452,14 +1452,14 @@ private Optional<AnchorPoint> tryLoadingAnchorPointFromInitialState(
       initialAnchor =
           attemptToLoadAnchorPoint(
               networkConfiguration.getNetworkBoostrapConfig().getInitialState());
-    } catch (final InvalidConfigurationException e) {
+    } catch (final InvalidConfigurationException ex) {
       final StateBoostrapConfig stateBoostrapConfig =
           networkConfiguration.getNetworkBoostrapConfig();
       if (stateBoostrapConfig.isUsingCustomInitialState()
           && !stateBoostrapConfig.isUsingCheckpointSync()) {
-        throw e;
+        throw ex;
       }
-      STATUS_LOG.warnFailedToLoadInitialState(e.getMessage());
+      STATUS_LOG.warnFailedToLoadInitialState(ex);
```

Comment:
would prefer no stacktrace on status log...

Reply:
I've changed the method to only print the log the message but the param we're passing could remain the throwable I think